crossplane/__init__.py

```diff
@@ -2,8 +2,9 @@
 from .parser import parse
 from .lexer import lex
 from .builder import build
+from .objects import xmap, ximport
 
-__all__ = ['parse', 'lex', 'build']
+__all__ = ['parse', 'lex', 'build', 'xmap', 'ximport']
```

Comment:
I think the names of these functions and what they do are counter-intuitive. Like without reading the docstring, I'd have no idea what `crossplane.ximport` could possibly mean.

Reply:
don't worry about this for now, I'll change the API around after I accept your changes.

Reply:
So the idea here was "x" stands for "cross" (hehe get it).  But mostly I thought of "map" as mapping of JSON to Python objects (much akin to how SQLAlchemy is an ORM) and "import" is...well what I thought was the best description.  Feel free to suggest different names, but I thought they were the most descriptive (I went back and forth a lot) and I didn't want to override Python keywords.

Reply:
What you are submitting isn't really like an ORM or a map... It's just the same thing as parse, except the output will be nested Python objects instead of dicts/lists. And if we are adding the option to use nested objects to the module, I think it's important to treat it like this is the main way it should be used.

For that reason, I'll probably create something like an `NginxConfigParser` class that has a `parse` method that takes a filename and a `from_dict` method or something that takes a python dict/list payload that both create the nested `NginxConfig` objects. I'll also keep the `parse` function for backwards compatibility of course.